### PR TITLE
Update convert-user-mailbox-to-shared-mailbox.md

### PR DESCRIPTION
**Pull request by ninobb**
Removing the note about the new admin portal, because the steps are the same in the old and new M365 admin portal. Adding a step about password reset and also adding a note as to why this is important.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@cmcatee-MSFT - PR is ready for final review.  

Thank you.

cc: @kenwith

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -51,13 +51,15 @@ When you convert a user's mailbox to a shared mailbox, all of the existing email
...
 1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.
 
 2. Select the name of the user whose mailbox you want to convert.
 
+3. Reset the user's password.
+
+> [!NOTE]
+> While it is not required to reset the user's password during the conversion of mailbox type, it is important to understand that unless the password is reset, **the original username and password will continue working** after the mailbox is converted.
+
 3. On the **Mail** tab, under **More actions**, select **Convert to shared mailbox**.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]: 4. On the **Mail** tab, under **More actions**, select **Convert to shared mailbox**.

Comment (cmcatee-MSFT) [reply Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]: > It's not required to reset the user's password during mailbox conversion. However, if the password is not reset, the original username and password continue to work after the mailbox conversion is finished.

**Comment by pebaum** [comment]
@ninobb can you please make the suggested changes to your edit and resubmit? thanks!

**Comment by cmcatee-MSFT** [comment]
@ninobb The changes I suggested need to be made to the file before I can approve the PR. Thanks.

**Pull request merged**